File: app/src/main/java/com/roy93group/cpuinfo/widgets/swiperv/listener/SwipeSwitchListener.java
package com.roy93group.cpuinfo.widgets.swiperv.listener;

import com.roy93group.cpuinfo.widgets.swiperv.SwipeMenuLayout;

public interface SwipeSwitchListener {

    void beginMenuClosed(SwipeMenuLayout swipeMenuLayout);

    void beginMenuOpened(SwipeMenuLayout swipeMenuLayout);

    void endMenuClosed(SwipeMenuLayout swipeMenuLayout);

    void endMenuOpened(SwipeMenuLayout swipeMenuLayout);

}

File: app/src/main/java/com/roy93group/cpuinfo/widgets/swiperv/listener/SwipeSwitchListener.kt
package com.roy93group.cpuinfo.widgets.swiperv.listener

import com.roy93group.cpuinfo.widgets.swiperv.SwipeMenuLayout

interface SwipeSwitchListener {
    fun beginMenuClosed(swipeMenuLayout: SwipeMenuLayout?)
    fun beginMenuOpened(swipeMenuLayout: SwipeMenuLayout?)
    fun endMenuClosed(swipeMenuLayout: SwipeMenuLayout?)
    fun endMenuOpened(swipeMenuLayout: SwipeMenuLayout?)
}
